### PR TITLE
fix(dedicated-datacenter): handle minimum hosts

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
block user from backup offer when hosts is less than 2 and provide a button to order new hosts

ref: MANAGER-4571

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/controller.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { BACKUP_MINIMUM_HOST_COUNT } from '../backup.constants';
+
+export default class {
+  constructor() {
+    this.minimumHosts = BACKUP_MINIMUM_HOST_COUNT;
+  }
+}
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/controller.js]: Suggestion: You can get rid of the controller file, by passing the constant in the routing, and pass it into component bindings.

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "dedicatedCloud_datacenter_backup_minimum_hosts_description": "Pour activer la sauvegarde veeam, Il est nécessaire que votre datacentre dispose d'au minimum de {{minimumHosts}} hosts.",
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/translations/Messages_fr_FR.json]:
```suggestion
  "dedicatedCloud_datacenter_backup_minimum_hosts_description": "Pour activer la sauvegarde veeam, il est nécessaire que votre datacentre dispose d'au minimum {{minimumHosts}} hosts.",
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/controller.js (hunk 1 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/controller.js]: Good one @JeremyDec . Thanks. This is done :)

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/translations/Messages_fr_FR.json (hunk 2 above)
Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/translations/Messages_fr_FR.json]: This is changed :)

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "dedicatedCloud_datacenter_backup_minimum_hosts_description": "Pour activer la sauvegarde veeam, Il est nécessaire que votre datacentre dispose d'au minimum {{minimumHosts}} hosts.",
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/minimumHosts/translations/Messages_fr_FR.json]:
```suggestion
  "dedicatedCloud_datacenter_backup_minimum_hosts_description": "Pour activer la sauvegarde Veeam, Il est nécessaire que votre datacentre dispose d'au minimum {{minimumHosts}} hosts.",
```

**Pull request merged**